handsontable/src/plugins/formulas/formulas.js

```diff
@@ -367,6 +353,16 @@ export class Formulas extends BasePlugin {
     super.destroy();
   }
 
+  /**
+   * Update sheetName and sheetId properties.
+   *
+   * @param {string} [sheetName] The new sheet name.
+   */
+  updateSheetNameAndSheetId(sheetName) {
```

Comment:
Do you think this method can be useful to the user or is it better to leave it private? If private, then I would use the private method `#updateSheetNameAndSheetId`. If not then it's good.

Reply:
It was intended to be a typical helper method. I marked it as private.